packages/scandipwa/src/component/MyAccountMyWishlist/MyAccountMyWishlist.type.ts

```diff
@@ -9,26 +9,32 @@
  * @link https://github.com/scandipwa/scandipwa
  */
 
+import { Location } from 'history';
+
+import { WishlistPageInfo } from 'Store/Wishlist/Wishlist.type';
 import { IndexedWishlistProduct } from 'Util/Product/Product.type';
 
 export interface MyAccountMyWishlistContainerMapStateProps {
     wishlistItems: Record<string, IndexedWishlistProduct>;
+    wishlistPageInfo: WishlistPageInfo;
     isWishlistLoading: boolean;
     isMobile: boolean;
 }
 
 export interface MyAccountMyWishlistContainerMapDispatchProps {
     clearWishlist: () => void;
-    moveWishlistToCart: (sharingCode?: string) => Promise<void>;
+    moveWishlistToCart: (productsPerPage: number, sharingCode?: string) => Promise<void>;
     showPopup: <T>(payload: T) => void;
     showNotification: (message: string) => void;
     showError: (message: string) => void;
     removeSelectedFromWishlist: (options: string[]) => void;
+    updateWishlistProducts: (page: number, productsPerPage: number) => void;
 }
 
 export interface MyAccountMyWishlistContainerBaseProps {
     creatorsName: string;
     isEditingActive: boolean;
+    location: Location;
```

Comment:
Small hint, if using HOC withRouter you can extend props type with react-router special interface. It will include all types that HOC provides, not only location. But again, its just for future, out here better to use location from history util.